File: quinta_aula/primeiro.c
#include <stdio.h>

void main() {
    float notas[10], total;
    int i;

    for(i = 0; i < 10; i++) {
        printf("Digite a nota do aluno %d: ", i+1);
        scanf("%f", &notas[i]);
        total += notas[i] / 10;
    }
    
    printf("A média das notas dos alunos é de %f", total);
}

File: quinta_aula/primeiro.php
<?php

$notas = array();

for($i = 1; $i <= 10; $i++) {
    $n = readline("Digite a nota do aluno $i: ");
    array_push($notas, $n);
}

$soma = array_sum($notas) / count($notas);

echo "A média das notas dos alunos é de $soma";

?>